SpatialGDK/Source/SpatialGDK/Private/Utils/ActorGroupManager.cpp

```diff
@@ -72,3 +72,11 @@ FName UActorGroupManager::GetWorkerTypeForActorGroup(const FName& ActorGroup) co
 
 	return DefaultWorkerType;
 }
+
+bool UActorGroupManager::IsSameWorkerType(const AActor* ActorA, const AActor* ActorB)
+{
+	const FName& WorkerTypeA = GetWorkerTypeForClass(ActorA->GetClass());
```

Comment:
Worth adding a check that both Actors are not null

Reply:
Done